### PR TITLE
Add Channels sidebar button

### PR DESCRIPTION
**Pull request by matthew-src**
#### Summary

This PR documents the Add channels button that is in the Channels sidebar

### CODE REVIEW

**Comment by cwarnermm** [comment]
Great update, @matthew-src! The generated preview isn't showing your changes for some reason. Not sure why yet. Are you able to preview your changes locally? I'm curious to know how your new screenshot displays, and whether it displays bigger than anticipated.

**Comment by matthew-src** [comment]
Hi @cwarnermm, I do not have a local preview at the moment. I will see if I can fix it so it will work in the cloud preview environment.

**Comment by cwarnermm** [comment]
@matthew-src - I suspect that our inability to see updated generated previews is a pipeline issue and not something our team can fix. I've logged a ticket with our Release Team to investigate further.

I can pull this PR down and build it locally to review the image size. I can do so tomorrow morning.

File: source/channels/join-leave-channels.rst [hunk 1]
```diff
@@ -16,11 +16,16 @@ To join a private channel, you need to be added to the channel by a member of th
 
   .. tab:: Desktop
 
-    1. Select the **+** symbol at the top of the channel sidebar, then select **Browse Channels**. 
+    1. Select the **Add channels** button in the channel sidebar, then select **Browse Channels**.
+     
+      .. image:: ../images/add-channels-button.png
+        :alt: You can browse channels using the Add channels button
```
Comment (cwarnermm) [inline source/channels/join-leave-channels.rst:22]: Please remove the red outline from this screenshot (to align with our style guide).

Also, I've confirmed in a local build that this image appears quite large compared to the next image on the same page. 

image

Please add the following line to resize the image directly following the ``alt`` line:

```
:width: 400
```

This line will resize the image to a max width of 400 pixels. Please apply these updates to both instances of the image (here and the join/leave page).

**Review by cwarnermm — COMMENTED** [review]
@matthew-src - I've built this PR locally to review, and have provided feedback inline.

File: source/channels/join-leave-channels.rst (hunk 1 above)
Comment (matthew-src) [reply source/channels/join-leave-channels.rst:22]: Ok, @cwarnermm. I will work on this next week.

**Comment by matthew-src** [comment]
Hi @cwarnermm, I made those commits. The screenshot is new, so it will probably look different in the preview.

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**